build/filter_test.go

```diff
@@ -230,21 +230,35 @@ var _ = Describe("Test filters utilities:", func() {
 	Describe("Test platform network filters", func() {
 		deployment := Deployment{}
 		deployment.PlatformNetworks = make([]*v1.PlatformNetwork, 0)
+		deployment.AddressPools = make([]*v1.AddressPool, 0)
 		coreNetworkFilter := NewCoreNetworkFilter()
-		var get_platform_network = func(nwk_type string) *v1.PlatformNetwork {
+		var get_platform_network = func(nwk_type string, associatedAddressPools []string) *v1.PlatformNetwork {
 			spec := v1.PlatformNetworkSpec{
-				Type: nwk_type,
+				Type:                   nwk_type,
+				AssociatedAddressPools: associatedAddressPools,
 			}
 			new_plat_nwk := v1.PlatformNetwork{}
 			new_plat_nwk.Spec = spec
 			return &new_plat_nwk
 		}
-		deployment.PlatformNetworks = []*v1.PlatformNetwork{
-			get_platform_network("oam"),
-			get_platform_network("mgmt"),
-			get_platform_network("admin"),
-			get_platform_network("storage")}
-
+		var get_associated_address_pool = func(network_ip string) *v1.AddressPool {
+			spec := v1.AddressPoolSpec{
+				Subnet: network_ip,
```

Comment:
It's fine to hardcode some IP here, doesn't really matter for this filter.

Reply:
Done